tests/bot/cogs/moderation/test_utils.py

```diff
@@ -377,5 +377,5 @@ def setUp(self):
 
         actual = await utils.post_infraction(self.ctx, self.user, "mute", "Test reason")
         self.assertEqual(actual, "foo")
-        self.bot.api_client.post.assert_awaited_with("bot/infractions", json=payload)
+        self.bot.api_client.post.assert_has_awaits([call("bot/infractions", json=payload)] * 2)
```

Comment:
I was thinking of just using call count originally but this is an even better way of doing it.